packages/validator/test/unit/utils/params.test.ts

```diff
@@ -3,13 +3,20 @@ import {ChainConfig, chainConfigToJson} from "@lodestar/config";
 import {chainConfig} from "@lodestar/config/default";
 import {networksChainConfig} from "@lodestar/config/networks";
 import {NotEqualParamsError, assertEqualParams} from "../../../src/util/params.js";
-import {lighthouseHoleskyConfig, nimbusHoleskyConfig, prysmHoleskyConfig, tekuHoleskyConfig} from "./interopConfigs.js";
+import {
+  grandineHoodiConfig,
+  lighthouseHoodiConfig,
+  nimbusHoodiConfig,
+  prysmHoodiConfig,
+  tekuHoodiConfig,
+} from "./interopConfigs.js";
 
 const testCases: {name: string; items: [ChainConfig, Record<string, string>]}[] = [
```

Comment:
you will need to update this type and use `SpecJson` from `@lodestar/config`


```suggestion
const testCases: {name: string; items: [ChainConfig, SpecJson]}[] = [
```